manifests/apache.pp

```diff
@@ -81,7 +81,7 @@
       ssl_cert                   => $pulp::https_cert,
       ssl_key                    => $pulp::https_key,
       ssl_chain                  => $pulp::https_chain,
-      ssl_ca                     => $pulp::https_ca_cert,
+      ssl_ca                     => pick($pulp::https_ca_cert, $pulp::ca_cert),
```

Comment:
Given the repetition of this, I'm wondering if in `init.pp` there should be a `$real_https_ca_cert`. Both are ugly.

Reply:
We could do that. But I suspect we need the workaround in `manifests/child/apache.pp` anyway because of some special inheritance scenario. I would prefer to keep the logic close to the location where the variable is used.